packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilterDialog.tsx

```diff
@@ -163,11 +197,22 @@ function LoadedFilterDialogContent(props: LoadedFilterDialogContentProps) {
     [buildFilter, descriptor, filteringProps.selectedClasses],
   );
 
+  const onResetHandler = () => {
+    filteringProps.selectedClasses = [];
+    filteringProps.onSelectedClassesChanged([]);
+    rootGroup.items.forEach((item) => {
+      actions.removeItem([item.id]);
+    });
```

Comment:
I don't think this is a good approach to clear filter. I think we need to add `clear` actions to `FilterBuilderActions`

Reply:
Agree, especially as this action is also going to be needed when class selection changes